Makefile

```diff
@@ -27,10 +27,10 @@ ISTIO_DOCKER_HUB ?= docker.io/istio
 export ISTIO_DOCKER_HUB
 ISTIO_GCS ?= istio-release/releases/$(VERSION)
 ISTIO_URL ?= https://storage.googleapis.com/$(ISTIO_GCS)
-ISTIO_CNI_DOCKER_HUB ?= docker.io/istio
-export ISTIO_CNI_DOCKER_HUB
-ISTIO_CNI_DOCKER_TAG ?= master-latest-daily
-export ISTIO_CNI_DOCKER_TAG
+ISTIO_CNI_HUB ?= gcr.io/istio-release
+export ISTIO_CNI_HUB
+ISTIO_CNI_TAG ?= master-latest-daily
```

Comment:
Note to reviewers - this doesn't look correct, but master-latest-daily is the only CNI tagged image in the gcr.io HUB: 
<img width="321" alt="screen shot 2019-02-21 at 5 24 16 am" src="https://user-images.githubusercontent.com/755849/53168924-0fa12c00-3599-11e9-8eae-445a60ad5bec.png">

Reply:
There is a bit a of a problem here that I need to sort out with @hklai - at the least this should be getting updated nightly like the other images (e.g. pilot) since CNI wasn't in any of the other releases it's not surprising there there is only master-latest-daily,  but it is badly out of date.

Reply:
@hklai left the team, contact me instead.

Reply:
Actually, the image is "install-cni" and it's updated regularly for master but not tagged for release-1.1 yet:  https://console.cloud.google.com/gcr/images/istio-release/GLOBAL/install-cni?gcrImageListsize=30

There are no compatibility issues with Istio release-1.1 and CNI master and we gate istio-cni PRs on both Istio master (nightly builds) and Istio release-1.1 latest.  IMO, we should merge this fix as is and update the CNI tag in another PR when the CNI release-1.1 tag process fix is available.